homeassistant/components/zwave/__init__.py

```diff
@@ -195,7 +194,7 @@ def get_config_value(node, value_index, tries=5):
 @asyncio.coroutine
 def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
     """Generic Z-Wave platform setup."""
-    if discovery_info is None or NETWORK is None:
+    if discovery_info is None or hass.data[ZWAVE_NETWORK] is None:
```

Comment:
Actually, this cannot happen because zwave is a dependency and it will only set up a platform if the main component set up correctly.